volatility3/framework/plugins/windows/modscan.py

```diff
@@ -25,7 +25,7 @@ def get_requirements(cls):
         return [
             requirements.ModuleRequirement(name = 'kernel', description = 'Windows kernel',
                                                      architectures = ["Intel32", "Intel64"]),
-            requirements.VersionRequirement(name = 'poolerscanner',
+            requirements.VersionRequirement(name = 'poolscanner',
```

Comment:
This is fine, it's a textual name and isn't really used other than for storage in the config tree.  Should be no issues with changing it.  5:)